ascii.c:197:12: error: conflicting types for ‘strcasecmp’

--- origsrc/ascii-3.10/ascii.c	2010-10-19 00:20:54.000000000 -0500
+++ src/ascii-3.10/ascii.c	2010-10-20 22:19:03.447390300 -0500
@@ -194,6 +194,7 @@ static void speak(unsigned int ch)
     (void) putchar('\n');
 }
 
+#ifndef stricmp
 static int stricmp(char	*s, char *t)
 /* case-blind string compare */
 {
@@ -201,6 +202,7 @@ static int stricmp(char	*s, char *t)
 	s++, t++;
     return (int)(*t - *s);
 }
+#endif
 
 static void ascii(char *str)
 {
