--- origsrc/ascii-3.12/Makefile	2012-11-23 00:57:11.000000000 -0600
+++ src/ascii-3.12/Makefile	2013-05-05 23:00:21.662017800 -0500
@@ -7,7 +7,7 @@ CFLAGS = -O
 all: ascii ascii.1
 
 ascii: ascii.c splashscreen.h nametable.h
-	$(CC) -DREVISION=$(VERS) ascii.c -o ascii
+	$(CC) $(CFLAGS) -DREVISION=$(VERS) ascii.c -o ascii
 
 splashscreen.h: splashscreen
 	sed <splashscreen >splashscreen.h -e 's/\\/\\\\/g' -e 's/"/\\"/g' -e 's/.*/P("&");/' 
